buildpipeline/Core-Setup-PortableLinux-x64-master.json

```diff
@@ -0,0 +1,154 @@
+{
+  "build": [
+    {
```

Comment:
I've discovered a way to run the command in a basic Run Command build task, by using `bash -c`, so I've updated the PR to use that approach.